src/main/java/org/opentripplanner/raptor/rangeraptor/DefaultRangeRaptorWorker.java

```diff
@@ -269,7 +270,12 @@ private void addAccessPaths(Collection<RaptorAccessEgress> accessPaths) {
     }
 
     for (RaptorAccessEgress it : accessPaths) {
-      transitWorker.setAccessToStop(it, iterationDepartureTime);
+      int departureTime = calculator.departureTime(it, iterationDepartureTime);
```

Comment:
This will be wrong in the heuristic/noWait case, since departureTime now would include waiting time?

Reply:
Now set in the state, not here.